monadic-rx/shared/src/main/scala/mhtml/rx.scala

```diff
@@ -293,6 +326,8 @@ object Var {
   /** Create a `Var` from an initial value. */
   def apply[A](initialValue: A): Var[A] =
     new Var[A](Some(initialValue), _ => Cancelable.empty)
+  def apply[A](initialValue: A, allowMultiMutations: Boolean): Var[A] =
```

Comment:
How about instead of using a simple Boolean, we use an implicit parameter that wraps a Boolean? - then the user can specify how they want their vars to be handled by default, and not rely on a default defined by the mhtml project.